frontend/src/concepts/roleBinding/RoleBindingPermissionsTableSection.tsx

```diff
@@ -13,23 +13,34 @@ import {
 import { RoleBindingKind } from '~/k8sTypes';
 import HeaderIcon from '~/concepts/design/HeaderIcon';
 import { ProjectObjectType } from '~/concepts/design/utils';
-import ProjectSharingTable from './ProjectSharingTable';
-import { ProjectSharingRBType } from './types';
+import RoleBindingPermissionsTable from './RoleBindingPermissionsTable';
+import { RoleBindingPermissionsRBType, RoleBindingPermissionsRoleType } from './types';
 
-export type ProjectSharingTableSectionAltProps = {
+export type RoleBindingPermissionsTableSectionAltProps = {
```

Comment:
Why `Alt` here?

Reply:
idk, that's just the previous name with my ProjectSharing->RoleBindingPermissions swap

Reply:
Fair enough